src/compass/utils/lut.py

```diff
@@ -131,7 +172,13 @@ def compute_geocoding_correction_luts(burst, dem_path,
         List of numpy.ndarray containing SET in slant range and azimuth directions
         in meters. These corrections need to be added to the slC tagged azimuth
         and slant range times.
+
+    [wet_los_tropo, dry_los_tropo]: list, np.ndarray
```

Comment:
```suggestion
    [wet_los_tropo, dry_los_tropo]: list[np.ndarray]
```
This is less ambiguous. Also update SET corrections above the same way.